tests/test_eb.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import amrex.space3d as amr
+
+
+@pytest.mark.skipif(not amr.Config.have_eb, reason="Requires -DAMReX_EB=ON")
+def test_makeEBFab():
+    pass
+
+    # TODO:
+    # EB2_Build(...)
+    # makeEBFabFactory(...)
```

Comment:
@WeiqunZhang do you like to finalize your PR by adding a test that uses your new functionality, so we have test coverage?
```suggestion
    ... = EB2_Build(...)
    ... = makeEBFabFactory(...)
```

Reply:
Yes, I will do it.